erpnext/accounts/report/financial_statements.py

```diff
@@ -265,22 +265,30 @@ def add_to_list(parent, level):
 
 	return filtered_accounts, accounts_by_name, parent_children_map
 
+#added by ahmad ragheb
+#date 16/1/2017
+
 def sort_root_accounts(roots):
 	"""Sort root types as Asset, Liability, Equity, Income, Expense"""
 
 	def compare_roots(a, b):
-		if a.report_type != b.report_type and a.report_type == "Balance Sheet":
-			return -1
-		if a.root_type != b.root_type and a.root_type == "Asset":
-			return -1
-		if a.root_type == "Liability" and b.root_type == "Equity":
-			return -1
-		if a.root_type == "Income" and b.root_type == "Expense":
-			return -1
-		return 1
+		# if a.report_type != b.report_type and a.report_type == "Balance Sheet":
```

Comment:
please don't keep commented code.

Also this feature should work with / without account numbers